emissionsapi/web.py

```diff
@@ -67,7 +68,7 @@ def entrypoint():
     production, but only during the development.
     """
     logger.info("Starting the Connexion Debug Server")
-    app.run()
+    app.run(host=config('host') or '127.0.0.1')
```

Comment:
Listen only locally is the way to go, but do we need to be able to configure the development server? IMHO 127.0.0.1 hardcoded is enough. If you want to deploy it, you should use gunicorn or wsgi anyway.

Reply:
Fair enough. I guess I was just anticipating complains ;-)
But I've added a second commit changing this so we can easily revert to this if we need to.